var/spack/repos/builtin/packages/py-h5sh/package.py

```diff
@@ -10,11 +10,11 @@ class PyH5sh(PythonPackage):
     """Shell-like environment for HDF5."""
 
     homepage = "https://github.com/sethrj/h5sh"
-    url      = "https://github.com/sethrj/h5sh/archive/v0.1.1.tar.gz"
+    pypi     = "h5sh/h5sh-0.1.1.tar.gz"
 
     maintainers = ['sethrj']
 
-    version('0.1.1', sha256='111989d8200d1da8e150aee637a907e524ca0f98d5005a55587cba0d94d9c4a0')
+    version('0.1.1', sha256='ccd8fed532d479d297baef044265100a4fb9cd119bce6f43270f2ee9f63a2183')
 
     depends_on('py-setuptools', type=('build', 'run'))
     depends_on('py-h5py', type=('build', 'run'))
```

Comment:
Can you add the version requirements to h5py and numpy?

Reply:
Done.